hologram_dashboard/fetcher/proto/hologram_availability.proto

```diff
@@ -0,0 +1,36 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+syntax = "proto2";
+package wireless_android_play_analytics;
+import "fetcher/fetcher_lib/enums.proto";
+
+message HologramDataAvailability {
+  // Specification about which data source is currently accessed
+  optional DataType data_source = 1;
+
+  // Specification about the status of the data source
+  message DataSourceDetail {
+      optional int64 date = 1;
+      optional StatusType status = 2;
+  }
+  
```

Comment:
nit: move the DataSourceDetail out?

message DataSourceDetail {
}

message HologramDataAvailability {
}

Reply:
Done